pom.xml

```diff
@@ -66,6 +66,7 @@
         <aws.sdk.version>1.11.75</aws.sdk.version>
         <cassandra.driver.version>3.2.0</cassandra.driver.version>
         <encoding>UTF-8</encoding>
+        <jackson.version>2.8.9</jackson.version>
```

Comment:
* add comment: `to fix a vulnerability, we use jackson 2.8.9 instead of 2.8.4 which would match jersey 2.25`
* move `jersey-media-json-jackson` version to a new property `jersey.version`